### PR TITLE
show codes set from local storage and persist

### PR DESCRIPTION
**Pull request by ngprnk**
# Changelog 
Peristed showcodes to `localstorage` 

# How to test ? 
1. goto `localhost:3000/home?showcodes=true`
2. hover over hamburger menu it should display the test id or any other components which has testid

### CODE REVIEW

File: src/views/components/tooltip/testid-tooltip/TestIdTooltip.js [hunk 1]
```diff
@@ -2,32 +2,51 @@ import React, { Component } from 'react';
...
+    let isDebugMode;
+
+    if ( typeof window === 'object' ) {
+      isDebugMode = window.originalQueryParams.showcodes;
+    }
+
+    const code = await Storage.get( 'showcodes' );
+
+    if ( code === true || code === 'true' ) {
+      isDebugMode = true;
```
Comment (DevJS) [inline src/views/components/tooltip/testid-tooltip/TestIdTooltip.js]: isDebugMode = (code === true || code === 'true')

File: src/views/components/tooltip/testid-tooltip/TestIdTooltip.js [hunk 2]
```diff
@@ -2,32 +2,51 @@ import React, { Component } from 'react';
...
     children: node,
+  };
+
+  // persist the showcodes on localstorgae
+  async persistCodes() {
+    const { children } = this.props;
+    let isDebugMode;
+
+    if ( typeof window === 'object' ) {
+      isDebugMode = window.originalQueryParams.showcodes;
```
Comment (DevJS) [inline src/views/components/tooltip/testid-tooltip/TestIdTooltip.js]: `showCodes` cammel case?

Comment (ngprnk) [reply src/views/components/tooltip/testid-tooltip/TestIdTooltip.js]: it was originally all low case, i'm keeping the compatibility

File: src/views/components/tooltip/testid-tooltip/TestIdTooltip.js (hunk 1 above)
Comment (ngprnk) [reply src/views/components/tooltip/testid-tooltip/TestIdTooltip.js]: also need to set it false.

**Pull request closed without merging**